nixos/modules/services/misc/jellyfin.nix

```diff
@@ -49,56 +49,61 @@ in
       after = [ "network.target" ];
       wantedBy = [ "multi-user.target" ];
 
+      # This is mostly follows: https://github.com/jellyfin/jellyfin/blob/master/debian/jellyfin.service
```

Comment:
Did you mean https://github.com/jellyfin/jellyfin/blob/master/fedora/jellyfin.service (fedora instead of debian)? I got confused because following the link I saw no hardening

Reply:
Looks like they dropped stuff again from debian.